docker/docker-compose.host.yml

```diff
@@ -0,0 +1,34 @@
+version: '3'
+
+services:
+  postgres:
+    network_mode: host
+    healthcheck:
+      test: ["CMD", "pg_isready", "-d", "postgres://osrd:password@localhost:5432/osrd"]
+
+  redis:
+    network_mode: host
+
+  core:
+    network_mode: host
+    environment:
+      CORE_PORT: 8080
+      MIDDLEWARE_BASE_URL: "http://localhost:8090"
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8080/health"]
+
+  editoast:
```

Comment:
We could override `ports` to avoid warning running this docker-compose file.

Reply:
Overriding `ports` with something like `ports: []` does not prevent the warning. Do you have something else in mind ?

Reply:
Never mind, I thought it would work.